program_info/genicons.sh

```diff
@@ -67,7 +67,4 @@ else
 fi
 
 # replace icon in themes
-for dir in ../launcher/resources/*/scalable
-do
-    cp -v org.prismlauncher.PrismLauncher.svg "$dir/launcher.svg"
-done
+cp -v org.prismlauncher.PrismLauncher.svg "../launcher/resources/multimc/scalable/launcher.svg"
```

Comment:
and on windows where bash/sh can't run?

Reply:
/shrug

this works best on macOS anyway, as we need some macOS tools

Reply:
this pr specifically hasn't created any issues, so resolving this